app/uk/gov/hmrc/organisationsmatchingapi/services/MatchedService.scala

```diff
@@ -17,25 +17,30 @@
 package uk.gov.hmrc.organisationsmatchingapi.services
 
 import com.google.inject.Inject
-import uk.gov.hmrc.organisationsmatchingapi.domain.models.{CtMatch, MatchNotFoundException, SaMatch}
+import uk.gov.hmrc.organisationsmatchingapi.domain.models.{CtMatch, MatchNotFoundException, SaMatch, UtrMatch}
 
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
-import scala.concurrent.Future.failed
+import scala.concurrent.Future.{failed, successful}
 
 class MatchedService @Inject()(cacheService: CacheService) {
 
   def fetchCt(matchId: UUID)(implicit ec: ExecutionContext) =
     cacheService.fetch[CtMatch](matchId) flatMap {
-      case Some(entry) =>
-        Future.successful(entry)
-      case _ => failed(new MatchNotFoundException)
+      case Some(entry) => Future.successful(entry)
+      case _           => failed(new MatchNotFoundException)
     }
 
   def fetchSa(matchId: UUID)(implicit ec: ExecutionContext) =
     cacheService.fetch[SaMatch](matchId) flatMap {
-      case Some(entry) =>
-        Future.successful(entry)
-      case _ => failed(new MatchNotFoundException)
+      case Some(entry) => Future.successful(entry)
+      case _           => failed(new MatchNotFoundException)
+    }
+
+  def fetchMatchedOrganisationRecord(matchId: UUID)
+                                    (implicit ec: ExecutionContext) =
+    cacheService.fetch[UtrMatch](matchId) flatMap {
+      case Some(utrMatch) => successful(UtrMatch(utrMatch.matchId, utrMatch.utr))
```

Comment:
I would either move all of these to be successfull(...) and failed(...) or all to be Future.successfull(...) and Future.failed(...) (my preference is the secons as its clear when reading). Consistency in file helps reduce how much you need to think about it, took me a while to realise where these functions were coming from.